### ISSUE TITLE
wrong link in documentation

### ISSUE THREAD

**Issue by ghost** [uyuni-project/uyuni#3280]
# Additional Information

_The following information is very important in order to help us to help you. Omission of the following details cause delays or could receive no attention at all._

## Version of Uyuni Server and Proxy (if used)
`zypper info Uyuni-Server-release` _and_ `zypper info Uyuni-Proxy-release` _(if used)_
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : Uyuni Server Stable
Name           : Uyuni-Server-release
Version        : 2021.01-107.1.uyuni
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : Uyuni-Server-release-2021.01-107.1.uyuni.src
Summary        : Uyuni Server

## Details about issue
_Include as many details as you can. The procedure you are following,  the commands you are running, logs (specially from_ `/var/log/rhn/`_)..._
_In general: all that can help reproducing the issue or understanding it_

at Admin -> Manager Configuration -> Monitoring. Right hand side the box "Server Monitoring". The link "documentation" is wrong:  is /docs/en/**suse-manager**/administration/monitoring.html should be /docs/en/**uyuni**/administration/monitoring.html

Regards,
Oliver

**Comment by agraul** [comment]
@Loquacity I can't transfer this to uyuni-docs, but I think that is the correct place for this issue.

**Comment by Loquacity** [comment]
> @Loquacity I can't transfer this to uyuni-docs, but I think that is the correct place for this issue.

Hi Alexander, I can move the issue into our repo, but if I'm reading OP correctly, then it's a problem in the UI, not in the docs themselves.

**Comment by agraul** [comment]
You're right, I didn't read the description correctly :see_no_evil:. Sorry for the noise @Loquacity

**Comment by sbluhm** [comment]
@mrstigmata , can you please check on master or the next release if this is still relevant? I have done some changes in that area to get the documentation working for me. It might have fixed it for you too.

**Comment by mrstigmata** [comment]
@sbluhm the behavior is still the same (-> /docs/en/**suse-manager**/administration/monitoring.html instead of /docs/en/**uyuni**/administration/monitoring.html)

**Comment by sbluhm** [comment]
@ghost, I just had a look on my current system (master code base) and the link was correct (Uyuni). Can you please check again?

**Comment by nodeg** [comment]
This is fixed as of now and points to `/docs/en/administration/monitoring.html`
